components/combobox/combobox.jsx

```diff
@@ -445,12 +527,11 @@ class Combobox extends React.Component {
 	/**
 	 * Menu open/close and sub-render methods
 	 */
-
 	handleClickOutside = (event) => {
 		this.handleRequestClose(event, {});
 	};
 
-	handleClose = (event) => {
+	handleClose = (event, { trigger } = {}) => {
```

Comment:
@interactivellama I wanted to add a way to cancel, is this OK?

Reply:
Yes, mimicking what is in Popover is good.